common/control_files/vrouter-functions.sh

```diff
@@ -74,3 +76,137 @@ function insert_vrouter() {
     fi
     return 0
 }
+
+#
+# For kernel vRouter we use instead insert_vrouter() function above
+#
+function vrouter_dpdk_start() {
```

Comment:
I understand this is the reason for code duplication in other files like vrouter-dpdk-post-start.sh for example. We could easily eliminate that by introducing common insert_vrouter function that would call routines for DPDK or kernel module here just by looking into the config. I think it is better approach as it eliminates much of code duplication.